src/amo/css/AddonReview.scss

```diff
@@ -1,52 +1,51 @@
 @import "~amo/css/inc/vars";
 
-.AddonReview {
-  text-align: center;
-  margin: auto;
-  width: 90%;
+.AddonReview .Card-contents {
+  padding-left: 0;
+  padding-right: 0;
+  overflow: hidden;
+}
+
+.AddonReview-form-input {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 0 10px;
 }
 
 .AddonReview-header {
-  font-size: $font-size-l;
+  font-size: $font-size-m + 8px;
```

Comment:
This seems odd, makes me wonder why we have the variable. Was this for testing or is the large too large?